src/cata_tiles.cpp

```diff
@@ -1944,7 +1945,7 @@ bool cata_tiles::draw_from_id_string( std::string id, TILE_CATEGORY category,
 
     // make sure we aren't going to rotate the tile if it shouldn't be rotated
     if( !display_tile.rotates ) {
-        rota = 0;
+        //rota = 0;
```

Comment:
Shouldn't commit commented out code. Also, commenting out the entire contents of the conditional removes need for conditional. Should delete the conditional, and its comment.

Reply:
> I like the idea! +1 However the string compares are unnecessary for the job.

thank you for the kind words and the advices on the code, it is much appreciated.
i will try to do it.